organizers/2018-04-10.html

```diff
@@ -0,0 +1,150 @@
+
+<section>
+    <h1>The London Django Meetup Group</h1>
+    <h2>April 10th 2018</h2>
+</section>
+
+<section>
+    <h1>Hello, I'm Craig Anderson!</h1>
+    <h2>👋😄</h2>
+</section>
+
+<section data-background='#eee'>
+    <h1>Sponsorship</h1>
+    <p>Tonight's meetup is kindly sponsored by:</p>
+    <ul>
+        <li><strong>Festicket</strong></li>
+        <li><strong>Funding Options</strong></li>
+        <li><strong>Thread</strong></li>
+        <li><strong>Time Out</strong></li>
+    </ul>
+</section>
+
+<section data-background='#eee'>
+    <img data-src="/img/festicket.svg" style="width:40%" alt="Festicket" class="plain"></img>
+    <p>
+        <em>“Festival package travel startup. 50+ staff, high pay, ping pong + beer.”</em>
+    </p>
+</section>
+
+<section data-background='#eee'>
+    <img data-src="/img/funding-options-logo-black.svg" style="width:40%" alt="Funding Options" class="plain"></img>
+    <p>
+        <em>Funding Options is the free marketplace for business finance, using matching technology to help UK businesses access tens
+        of millions of pounds in finance every year.
+        </em>
+    </p>
+</section>
+
+<section data-background='#eee'>
+    <img data-src="/img/thread.png" style="width:10%" alt="Thread" class="plain"></img>
+    <p>
+        <em>We help people to feel happier and more self-confident, by making it easy to dress well, using stylists and data science. Dedicated to maintaining a great working environment, minimising politics and maximising impact.</em>
+    </p>
+</section>
+
+<section data-background='#eee'>
+    <img data-src="/img/timeout.png" style="width:40%" alt="Time Out" class="plain"></img>
+    <p>
+        <em>The trusted global platform that inspires and enables people to experience the best of the city. Operating in 107 cities, across 39 countries, it has an average monthly global audience reach of over 111 million across all platforms.</em>
+    </p>
+</section>
+
+<section style="text-align: left">
+    <h1>📋 Today's Schedule</h1>
+    <h2>🎺 Django News</h2>
+</section>
+
+<section style="text-align: left">
+    <h1 style="text-align: center;">📋 Today's Schedule</h1>
+    <h2 style="text-align: center;">Talks</h2>
+    <ol>
+        <li style="overflow: hidden;">
+            <img src="https://res.cloudinary.com/skillsmatter/image/upload/c_crop,g_custom/c_scale,w_96,h_96/v1522945378/jf2mwcxlcurnze6iieyt.png"
+                style="float:left; border-radius: 100%; width: 150px; height 150px; margin: 5px 50px;"> Adam Charnock
+                - Freelancing (and Contracting) for Developers
+        </li>
+        <li style="overflow: hidden;">
+            <img src="https://res.cloudinary.com/skillsmatter/image/upload/c_crop,g_custom/c_scale,w_96,h_96/v1488977535/scrxd00x1vvwijoildba.png"
+                style="float:left; border-radius: 100%; width: 150px; height 150px; margin: 5px 50px;"> Dan Palmer
+                - django-enumfield
+        </li>
```

Comment:
Just had a thought - this order should probably be reversed since Adam C's talk isn't to be recorded, it will allow the AV assistant to start packing up early if it's last